orchestrator/gorc/src/commands/tx/cosmos.rs

```diff
@@ -87,6 +115,105 @@ impl Runnable for SendToEth {
+                })
+                .await;
+                match res {
+                    Ok(val) => println!(
+                        "Asset {} has ERC20 representation {}",
+                        denom,
+                        val.into_inner().erc20
+                    ),
+                    Err(_e) => {
+                        println!(
+                            "Asset {} has no ERC20 representation, you may need to deploy an ERC20 for it!",
+                            denom.clone()
+                        );
+                        exit(1);
+                    } }
+                    let amount = Coin {
+                        amount,
+                        denom: denom.clone(),
+                    };
+                    let bridge_fee = Coin {
+                        denom: denom.clone(),
+                        amount: 1u64.into(),
+                    };
+                    let eth_dest: EthAddress = to_eth_addr.parse().unwrap();
+                    check_for_fee_denom(&denom, cosmos_address, &contact).await;
+
+                    let balances = contact
+            .get_balances(cosmos_address)
+            .await
+            .expect("Failed to get balances!");
+        let mut found = None;
+        for coin in balances.iter() {
+            if coin.denom == denom {
+                found = Some(coin);
+            }
+        }
+        println!("Cosmos balances {:?}", balances);
+
+        let times = if let Some (x)=self.times{
+            x
+        } else {
+            1
+        };
+
+        if found.is_none() {
+            panic!("You don't have any {} tokens!", denom);
+        } else if amount.amount.clone() * times.into() >= found.clone().unwrap().amount
+            && times == 1
+        {
+            if is_cosmos_originated {
+                panic!("Your transfer of {} {} tokens is greater than your balance of {} tokens. Remember you need some to pay for fees!", print_atom(amount.amount), denom, print_atom(found.unwrap().amount.clone()));
+            } else {
+                panic!("Your transfer of {} {} tokens is greater than your balance of {} tokens. Remember you need some to pay for fees!", print_eth(amount.amount), denom, print_eth(found.unwrap().amount.clone()));
+            }
+        } else if amount.amount.clone() * times.into() >= found.clone().unwrap().amount {
+            if is_cosmos_originated {
+                panic!("Your transfer of {} * {} {} tokens is greater than your balance of {} tokens. Try to reduce the amount or the --times parameter", print_atom(amount.amount), times, denom, print_atom(found.unwrap().amount.clone()));
+            } else {
+                panic!("Your transfer of {} * {} {} tokens is greater than your balance of {} tokens. Try to reduce the amount or the --times parameter", print_eth(amount.amount), times, denom, print_eth(found.unwrap().amount.clone()));
+            }
+        }
+
+        for _ in 0..times {
```

Comment:
I'm probably misunderstanding the use of this call, but why would you want to make the same transfer multiple times? Seems like it would just waste gas. IMO, even if there is a reason to do this, we should remove it from this interface, and let the user write a bash script or repeat the same command manually.